src/engine/Camera.ts

```diff
@@ -88,14 +87,6 @@ export class BaseCamera {
       }
    }
 
-   /**
```

Comment:
This is a breaking change, and should be documented in the changelog.

Reply:
Actually no, it was dead code, did nothing